src/Controls/tests/TestCases.HostApp/Issues/Issue25409.xaml.cs

```diff
@@ -0,0 +1,33 @@
+namespace Maui.Controls.Sample.Issues;
+
+[Issue(IssueTracker.Github, 25409, "UIButton CurrentImage can be set without crashing", PlatformAffected.iOS)]
+public partial class Issue25409 : ContentPage
+{
+	public Issue25409()
+	{
+		InitializeComponent();
+	}
+
+	private void OnCounterClicked(object sender, EventArgs e)
+	{
+#if IOS
+		((Action)(async () =>
+		{
+			var imageSource = new FileImageSource() { File = "shopping_cart.png" };
```

Comment:
Could the test be extended to:
- Set the image when the button is tapped.
- Remove the image, setting a null value when tapping again.
- Reset the image by tapping the button again (for example, using a counter, odd sets image, even removes it).
?